shared/fs/footer/container.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+import * as I from 'immutable'
+import * as Types from '../../constants/types/fs'
+import {compose, connect, setDisplayName, type Dispatch, type TypedState} from '../../util/container'
+import Footer, {type FooterProps} from './footer'
+import * as FsGen from '../../actions/fs-gen'
+
+type StateProps = {
+  transfers: I.Map<string, Types.TransferState>,
+}
+
+type OwnProps = {}
+
+type actionFactory = (p: Types.LocalPath) => () => void
+type DispatchProps = {
+  opener: actionFactory,
+  dismisser: actionFactory,
+}
+
+const mapStateToProps = (state: TypedState, ownProps: OwnProps) => ({
```

Comment:
if you're not using ownProps do not refer to it. it causes it to opt out of certain optimizations if you do that